### PR TITLE
[author] Add inputmask w/ npm auto-update using single package.json 

### PR DESCRIPTION
**Pull request by RobinHerbots**
# Pull request template for new library

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/RobinHerbots/Inputmask
* Official website (optional, not the repository): http://robinherbots.github.io/Inputmask
* NPM package url (optional): https://www.npmjs.com/package/inputmask
* License and its reference: https://github.com/RobinHerbots/Inputmask/blob/4.x/LICENSE.txt
* GitHub / Bitbucket popularity (required):
  * Count of watchers: https://github.com/RobinHerbots/Inputmask/watchers
  * Count of stars: https://github.com/RobinHerbots/Inputmask/stargazers
  * Count of forks: https://github.com/RobinHerbots/Inputmask/network/members

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [x] This lib was not found on cdnjs repo
* [x] Doesn't already exist / has duplicate issue and PR 
* [x] The lib has notable popularity
  * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [x] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.

### CODE REVIEW

**Comment by RobinHerbots** [comment]
@dakshshah96 , 

Can you point me in the right direction to fix the ci build?  I'm missing something in the package.json.

**Comment by RobinHerbots** [comment]
@sufuf3 , @extend1994 , 

#13126
The name of the jquery.inputmask has changed to inputmask 

Versions on npm continue under inputmask.  
Latest jquery.inputmask => 3.3.4
Latest inputmask => 4.0.6

File: ajax/libs/inputmask/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "inputmask",
+  "version": "4.0.6",
```
Comment (PeterDaveHello) [inline ajax/libs/inputmask/package.json]: version info should be removed as it's not added yet

File: ajax/libs/inputmask/package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "inputmask",
+  "version": "4.0.6",
+  "description": "Inputmask is a javascript library which creates an input mask.  Inputmask can run against vanilla javascript, jQuery and jqlite.",
+  "filename": "dist/min/jquery.inputmask.bundle.min.js",
```
Comment (PeterDaveHello) [inline ajax/libs/inputmask/package.json]: `dist/min/` should be removed

File: ajax/libs/inputmask/package.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  "npmName": "inputmask",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "**/*"
+      ]
+    }
+  ]
+}
```
Comment (PeterDaveHello) [inline ajax/libs/inputmask/package.json]: here we need a final newline char.

File: ajax/libs/inputmask/package.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  ],
+  "author": {
+    "name": "Robin Herbots",
+    "url": "https://github.com/RobinHerbots"
+  },
+  "license": "MIT",
+  "npmName": "inputmask",
+  "npmFileMap": [
+    {
+      "basePath": "",
```
Comment (PeterDaveHello) [inline ajax/libs/inputmask/package.json]: we should set basePath to `dist/min` or `dist` to fetch the needed files.

**Comment by PeterDaveHello** [comment]
Let me help squash and rebase commits first.

**Comment by PeterDaveHello** [comment]
It's weird that the first commit fb699ac79b4f336768050c3c218fad5cbc283c3a has no author name ... which makes the reabse keep failing:
```
fatal: empty ident name (for <robin@LAPTOP214.Reynaers.org>) not allowed
```

I'll do another further manualy fix for it, @RobinHerbots you may want to check your git ocnfigs ;)

**Comment by RobinHerbots** [comment]
> It's weird that the first commit [fb699ac](https://github.com/cdnjs/cdnjs/commit/fb699ac79b4f336768050c3c218fad5cbc283c3a) has no author name ... which makes the reabse keep failing:
> 
> ```
> fatal: empty ident name (for <robin@LAPTOP214.Reynaers.org>) not allowed
> ```
> I'll do another further manualy fix for it, @RobinHerbots you may want to check your git ocnfigs ;)

@PeterDaveHello , 

I did the commit from WSL on my work laptop, ... git wasn't really configured ;-)

**Comment by PeterDaveHello** [comment]
@RobinHerbots got it! Would you mind to fix this one?

```
==========ERROR==========
Found file with dos format: ajax/libs/inputmask/package.json, please use `dos2unix` to convert it to unix format!
==========ERROR==========
```

**Comment by RobinHerbots** [comment]
@PeterDaveHello, 

Ok, done!

**Comment by PeterDaveHello** [comment]
Here we go :tada: https://cdnjs.com/libraries/inputmask

**Pull request merged**